Rubrik/Rubrik.psd1

```diff
@@ -223,6 +224,7 @@
                    'Protect-RubrikTag',
                    'Protect-RubrikVApp', 
                    'Protect-RubrikVM',
+                   'Protect-RubrikVolumeGroup',
```

Comment:
Please update the Changelog file as well with all of your hard work!  Thanks so much!